modules/gitops-additional-steps-for-disconnected-clusters.adoc

```diff
@@ -0,0 +1,43 @@
+[id="additional-steps-disconnected-clusters_{context}"]
+= Integrating Keycloak with the OpenShift OAuth server in a disconnected cluster
+
+In a disconnected cluster, Keycloak communicates with OpenShift OAuth server through proxy. Follow the additional steps to integrate Keycloak with OpenShift OAuth server.
+
+.Procedure
+
+. Log in to the Keycloak pod:
++
+[source,yaml]
```

Comment:
```suggestion
[source,terminal]
```
For CLI commands, the code block descriptor should be `[source,terminal]` for syntax highlighting purposes. Please apply to the entire PR. See the OCP doc guidelines here: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output

Reply:
@bburt-rh This page is not included in the final output. so not making the changes now.